hoomd/md/PotentialBond.h

```diff
@@ -66,6 +66,12 @@ template<class evaluator, class Bonds> class PotentialBond : public ForceCompute
 
     //! Actually compute the forces
     virtual void computeForces(uint64_t timestep);
+
+    virtual Scalar energyDiff(unsigned int idx_a,
```

Comment:
Same as for `ForceCompute`. The mesh specific virtual methods should be in a mesh-specific base class.